### PR TITLE
Typos, README instruction and ROS 2 conventions

### PR DESCRIPTION
**Pull request by vmayoral**
Feel free to cherry-pick among the suggested changes if the naming convention adjustments are undesired.

### CODE REVIEW

File: concepts/index.rst [hunk 1]
```diff
@@ -16,7 +16,7 @@ Action Server
 Just as in ROS, action servers are a common way to control long running tasks like navigation.
 This stack makes more extensive use of actions, and in some cases, without an easy topic interface.
 It is more important to understand action servers as a developer in ROS 2.
-Some simple CLI examples can be found in the `ROS 2 documementation <https://docs.ros.org/en/rolling/Tutorials/Understanding-ROS2-Actions.html>`_.
+Some simple CLI examples can be found in the `ROS 2 documementation <https://docs.ros.org/en/rolling/Tutorials/Understanding-ROS 2-Actions.html>`_.
```
Comment (SteveMacenski) [inline concepts/index.rst:19]: You changed a URL

File: tutorials/docs/camera_calibration.rst [hunk 2]
```diff
@@ -55,7 +55,7 @@ Tutorial Steps
 
 |
 
- .. image:: images/Camera_Calibration/ROS2_topic_hz.png
+ .. image:: images/Camera_Calibration/ROS 2_topic_hz.png
```
Comment (SteveMacenski) [inline tutorials/docs/camera_calibration.rst:58]: You changed a filename

File: setup_guides/odom/setup_odom.rst [hunk 3]
```diff
@@ -79,7 +79,7 @@ We also need to install the ``gazebo_ros_pkgs`` package to simulate odometry and
  
   sudo apt install ros-<ros2-distro>-gazebo-ros-pkgs
 
-You can test if you have successfully set up your ROS 2 and Gazebo environments by following the instructions `here <http://gazebosim.org/tutorials?tut=ros2_installing&cat=connect_ros#TestingGazeboandROS2integration>`_. 
+You can test if you have successfully set up your ROS 2 and Gazebo environments by following the instructions `here <http://gazebosim.org/tutorials?tut=ros2_installing&cat=connect_ros#TestingGazeboandROS 2integration>`_.
```
Comment (SteveMacenski) [inline setup_guides/odom/setup_odom.rst:82]: You changed a URL

File: about/related_projects.rst [hunk 4]
```diff
@@ -8,7 +8,7 @@ This is a community maintained list of related repositories and projects to Navi
 +--------------------------------+------------------------+----------------------------------+
 |                   Name         |         Maintainer     |       Description                |
 +================================+========================+==================================+
-| `Navigation2`_                 | Steve Macenski         | Main ROS2 Navigation Stack       |
+| `Navigation2`_                 | Steve Macenski         | Main ROS 2 Navigation Stack       |
```
Comment (SteveMacenski) [inline about/related_projects.rst:11]: You must correct the alignment on the tables or it will render

File: README.md [hunk 5]
```diff
@@ -7,7 +7,7 @@ This folder holds the source and configuration files used to generate the
 Dependencies for Build: 
 * [Sphinx](https://www.sphinx-doc.org/en/master/usage/installation.html)
 * `sudo apt install python3-pip`
-* `pip3 install breathe==4.12.0 sphinx_rtd_theme sphinxcontrib-plantuml`
+* `pip3 install breathe==4.30.0 sphinx_rtd_theme sphinxcontrib-plantuml`
```
Comment (SteveMacenski) [inline README.md:10]: Don't change our requirements please

File: migration/Foxy.rst [hunk 6]
```diff
@@ -182,7 +182,7 @@ There's more this does, that that's the general information. See the package's `
 Costmap2D ``current_`` Usage
 ****************************
 
-In costmap2D, ``current_`` was used in ROS1 to represent whether a costmap layer was still enabled and actively processing data. It would be turned to ``false`` only under the situation that the expected update rate of a sensor was not met, so it was getting stale or no messages. It acts as a fail-safe for if a navigation sensor stops publishing.
+In costmap2D, ``current_`` was used in ROS to represent whether a costmap layer was still enabled and actively processing data. It would be turned to ``false`` only under the situation that the expected update rate of a sensor was not met, so it was getting stale or no messages. It acts as a fail-safe for if a navigation sensor stops publishing.
```
Comment (SteveMacenski) [inline migration/Foxy.rst:185]: Please revert all ROS's to ROS1's, I want that to be really explicit so there is no confusion.

**Review by SteveMacenski — CHANGES_REQUESTED** [review]

File: tutorials/docs/camera_calibration.rst (hunk 2 above)
Comment (vmayoral) [reply tutorials/docs/camera_calibration.rst:58]:
```suggestion
 .. image:: images/Camera_Calibration/ROS2_topic_hz.png
```

File: setup_guides/odom/setup_odom.rst (hunk 3 above)
Comment (vmayoral) [reply setup_guides/odom/setup_odom.rst:82]:
```suggestion
You can test if you have successfully set up your ROS 2 and Gazebo environments by following the instructions `here <http://gazebosim.org/tutorials?tut=ros2_installing&cat=connect_ros#TestingGazeboandROS2integration>`_. 
```

File: about/related_projects.rst (hunk 4 above)
Comment (vmayoral) [reply about/related_projects.rst]: Thanks and sorry, addressed at https://github.com/ros-planning/navigation.ros.org/pull/193/commits/c1672ad9bbc609a2dadcec8489e2b89b21d38645

File: README.md (hunk 5 above)
Comment (vmayoral) [reply README.md:10]: I didn't mean to mess with them, but I found that in OS X `make html` throws an error with `breathe==4.12.0` and only after updating to `breathe==4.30.0` things worked properly.

Do you still want me to revert the change?

File: migration/Foxy.rst (hunk 6 above)
Comment (vmayoral) [reply migration/Foxy.rst]: 2db7ee773a9637ab6dfb99805e0b20dd5a812143 cancels these.

File: README.md (hunk 5 above)
Comment (SteveMacenski) [reply README.md:10]: Or change it here as well where we build in CI https://github.com/ros-planning/navigation.ros.org/blob/master/.circleci/config.yml#L20 then if CI works, then we're all good. I just want to keep them consistent (which it doesn't look like they are right now anyway)

Comment (vmayoral) [reply README.md:10]: All right, updated CI at 060ad71d3d7549c1fe445125892603b408fc220d. Waiting for it to finish. I'll resolve this one if successful.

**Comment by SteveMacenski** [comment]
Seems to have failed to build

**Comment by vmayoral** [comment]
Yeap, reverting then. I'll bring things back to `4.12.0`. Including CI. Hopefully it'll succeed now.

**Comment by vmayoral** [comment]
@SteveMacenski it seems CI likes `4.9.1` and nothing more. Made it consistent also in the README. Ping me if there's anything left.

File: README.md (hunk 5 above)
Comment (vmayoral) [reply README.md]: ebbb012360ac9fe3cb1d35a009d5ed5fdba2d030

**Review by SteveMacenski — APPROVED** [review]
Thanks!

**Pull request merged**